src/vendor/wordpress/_wordpress.scss

```diff
@@ -111,6 +106,50 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: Image
+ * Styles for images inserted via Gutenberg blocks.
+ *
+ * 1. Since we are applying `margin-top` to CMS content via our staggerItems
+ *    utility, the default `margin-bottom` value on Gutenberg image blocks is
+ *    unnecessary, so we remove it here.
+ * 2. Setting a width of 100% prevents resized and centered images from breaking
+ *    layout when placed after a right-aligned image in a post.
+ * 3. Applies an opinionated wide-image setting that allows an image element to
+ *    spill slightly outsides its container at larger viewports.
+ */
+.wp-block-image {
+  margin-bottom: 0; /* 1 */
+  position: relative;
+
+  &.aligncenter {
+    width: 100%; /* 2 */
+  }
+
+  &.alignfull {
+    left: 50%;
+    margin-left: -50vw;
+    margin-right: -50vw;
+    right: 50%;
+    width: 100vw;
+  }
+
+  &.alignwide {
+    left: 50%;
+    margin-left: -50vw;
+    margin-right: -50vw;
+    right: 50%;
+    width: 100vw;
+    @media (min-width: breakpoint.$l) {
+      left: calc(12vw / -2); /* 3 */
+      margin-left: auto;
+      margin-right: auto;
+      right: auto;
+      width: calc(100% + 12vw); /* 3 */
```

Comment:
I think if you used `calc` you could simplify? (See earlier comment.)

Reply:
Agreed. I attempted to address this feedback in 6664d90